# 3 hyper-parameters of a squared exponential kernel
float32 var_f
float32 charLen
float32 var_n
